golang/Chapter05/model/client.go

```diff
@@ -0,0 +1,440 @@
+package model
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/go-redis/redis/v7"
+	uuid "github.com/satori/go.uuid"
+	"log"
+	"math"
+	"redisInAction/Chapter05/common"
+	"redisInAction/utils"
+	"reflect"
+	"sort"
+	"strconv"
+	"strings"
+	"time"
+)
+
+type Client struct {
+	Conn *redis.Client
+}
+
+func NewClient(conn *redis.Client) *Client {
+	return &Client{Conn: conn}
+}
+
+func (c *Client) LogRecent(name, message, severity string, pipeliner redis.Pipeliner) {
+	if severity == "" {
+		severity = "INFO"
+	}
+	destination := fmt.Sprintf("recent:%s:%s", name, severity)
+	message = time.Now().Local().String() + " " + message
+
+	if pipeliner == nil {
+		pipeliner = c.Conn.Pipeline()
+	}
+	pipeliner.LPush(destination, message)
+	pipeliner.LTrim(destination, 0, 90)
```

Comment:
Book reads 99 here, easy transcription mistake especially with a keyboard. Solid work so far!

Reply:
Fix it!